docs/source/release_notes.rst

```diff
@@ -3,6 +3,7 @@ Release Notes
 **Future Releases**
     * Enhancements
     * Fixes
+        * Fixed Data Checks errors :pr:`4274`
```

Comment:
This is a bit vague for a release note, users should be able to look at this and understand what changed. Can you move this to the `Documentation Changes` section and rephrase it to something along the lines of `Removed erroneous warnings from Data Checks User Guide page`?